modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,328 @@
+  - apiGroups:
+    - admissionregistration.k8s.io
+    resources:
+    - validatingwebhookconfigurations
+    - mutatingwebhookconfigurations
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    verbs:
+    - get
+    - list
+    - watch
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: <aggregated_apiserver_cluster_role>
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: <aggregated_apiserver_cluster_role>
+  subjects:
+  - kind: ServiceAccount
+    namespace: <namespace_name>
+    name: server
+----
+<1> Delegates authentication and authorization to the webhook server API.
+<2> Allows the admission webhook server to access cluster resource.
+<3> Enables the aggregated API server to create admission reviews.
+<4> Also enables the admission server to access cluster resource.
+<5> Role binding to read the configuration for terminating authentication.
+<6> Default cluster role and cluster role bindings for an aggregated API server.
+
+. Apply those RBAC rules to the cluster:
++
+----
+$ oc auth reconcile -f <YAML_filename>
+----
+
+. Create a new project to run the DaemonSet:
++
+----
+$ oc create namespace <namespace_name>
+----
+
+. Define a YAML configuration that is used to deploy a webhook as a DaemonSet server in a project. Restrict it to master nodes only:
++
+[source,yaml]
+----
+- apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+    namespace: <namespace_name>
+    name: server
+    labels:
+      server: "true"
+  spec:
+    selector:
+      matchLabels:
+        server: "true"
+    template:
+      metadata:
+        name: server
+        labels:
+          server: "true"
+      spec:
+        serviceAccountName: server
+        containers:
+        - name: <container_name>
+          image: <image_registry_username>/<image_path>:<tag>  <1>
+          imagePullPolicy: IfNotPresent
+          command:
+          - <container_commands>
+          ports:
+          - containerPort: <container_port>
+          volumeMounts:
+          - mountPath: /var/serving-cert
+            name: serving-cert
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: <container_port>
+              scheme: HTTPS
+        volumes:
+        - name: serving-cert
+          secret:
+            defaultMode: 420
+            secretName: server-serving-cert
+----
+<1> Points to a webhook server container image.
+
+. Define a secret for the service serving cert signer:
++
+[source,yaml]
+----
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    namespace: <namespace_name>
+    name: server-serving-cert
+  type: kubernetes.io/tls
+  data:  <1>
+    tls.crt: /etc/pki/CA/serving-server.<namespace_name>.svc.crt
+    tls.key: /etc/pki/CA/serving-server.<namespace_name>.svc.key
+----
+<1> References the signed webhook server key and certificate.
+
+. Declare a service account:
```

Comment:
I'd stick with Define instead of Declare. It's not a big deal to repeat the same verb if it's what's truly accurate.

Reply:
Resolved.